cs-office-365/src/main/java/io/cloudslang/content/office365/utils/InputsValidation.java

```diff
@@ -69,6 +85,25 @@
         return exceptionMessages;
     }
 
+    @NotNull
+    public static List<String> verifyListMessagesInputs(@Nullable final String userPrincipalName, @Nullable final String userId,
```

Comment:
A similar method already exist with name `verifyGetMessageInputs()`. They are very similar, so I would introduce a new method with name `verifyCommonInputs()` that will keep the common part and inside the other methods, call it and also make custom validations.